### ISSUE TITLE
UserAccelerometerEvent  are not received on some device

### ISSUE THREAD

**Issue by eliaweiss** [flutter/flutter#61741]
Testing on Android release 9 (SDK 28), LENOVO Lenovo TB-X505F

The UserAccelerometerEvent  are not received, I checked with the debuger:

this is my code

```
  streamSubscription = userAccelerometerEvents.listen((UserAccelerometerEvent event) {
     print('imhere');
```

in 

```
  Stream<dynamic> receiveBroadcastStream([ dynamic arguments ]) {
    final MethodChannel methodChannel = MethodChannel(name, codec);
    StreamController<dynamic> controller;
    controller = StreamController<dynamic>.broadcast(onListen: () async {
      binaryMessenger.setMessageHandler(name, (ByteData reply) async {
        if (reply == null) {
          controller.close();
        } else {
          try {
            controller.add(codec.decodeEnvelope(reply));
```

The line `controller.add(codec.decodeEnvelope(reply));` is reachable but the line `_listToUserAccelerometerEvent(event.cast<double>())` in 

```
/// Events from the device accelerometer with gravity removed.
Stream<UserAccelerometerEvent> get userAccelerometerEvents {
  if (_userAccelerometerEvents == null) {
    _userAccelerometerEvents = _userAccelerometerEventChannel
        .receiveBroadcastStream()
        .map((dynamic event) =>
            _listToUserAccelerometerEvent(event.cast<double>()));
  }
  return _userAccelerometerEvents;
}
```

and also the 'imhere' message never get printed.

It seems that from some reason, the stream is not working on old devices.

I have a complicated scenario, where i capture images from the camera process them, and use the Accelerometer in the same time.

Initially I thought that Accelerometer is not available, so I run the code on startup and set a flag, but strangely the code works on startup.
 It breaks only when I use the camera in parallel.

I assume it is a memory issue.

## Steps to Reproduce

1. Run `flutter create bug`.
2. Update the files as follows: ... 
3. ... 

**Expected results:** 

**Actual results:** 

<details>
  <summary>Logs</summary>

```
```

```
```

```
```

</details>

**Comment by VladyslavBondarenko** [comment]
Hi @eliaweiss 
could you provide a [complete minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) and your `flutter doctor -v`?

**Comment by darshankawar** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
